trappy/bare_trace.py

```diff
@@ -134,5 +134,9 @@ def add_parsed_event(self, name, dfr, pivot=None):
 
     def finalize_objects(self):
         for trace_class in self.trace_classes:
+            trace_class.tracer = self
             trace_class.create_dataframe()
             trace_class.finalize_object()
+
+    def generate_data_dict(self):
+        return None
```

Comment:
Then we should have this raise the exception and we should catch the exception and then implement the fallback. So that it can be more explicit